app/src/main/java/com/example/internetshop/presentation/viewModel/AuthenticationViewModel.kt

```diff
@@ -27,8 +28,11 @@ class AuthenticationViewModel @Inject constructor(
 
     fun getToken() {
         if (username.get().isNullOrEmpty() || password.get().isNullOrEmpty()) {
-            AuthenticationEvent.ToastAuthenticationEvent("Fill in all the fields!")
+            navEventLiveData.value =
```

Comment:
As i understand `navEventLiveData` handle not only navigation events now. Would be better to rename it into `events`